lib/tasks/events.rake

```diff
@@ -20,6 +20,10 @@ namespace :import do
 
     import_events_from_source(args[:calendar_id], from)
   end
+
+  task nuke_papertrail: :environment do
```

Comment:
maybe lets make this a bit more neutral lol. "purge_papertrail"?

Reply:
I did contemplate purge, yeah lol